src/services/driveStorageAdapter.js

```
@@ -0,0 +1,62 @@
+  constructor(googleDriveManager) {
+    this.gdm = googleDriveManager;
+  }
+
+  async create(data) {
+    const content = this._serializeData(data);
+    const res = await this.gdm.saveFile(
+      "appDataFolder",
+      `sls_${Date.now()}.json`,
+      content,
+    );
+    return res && res.id ? res.id : null;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)